File: Blobcode/inject/log.js
/**
 * Log an info message.
 * @param message The message to log.
 */
const info = (message) => {
    sendMessage("debugPrint", { message: `[INFO] ${message}` });
    console.log(`[INFO] ${message}`);
}

/**
 * Log an error message.
 * @param message The message to log.
 */
const error = (message) => {
    sendMessage("debugPrint", { message: `[ERROR] ${message}` });
    console.error(`[ERROR] ${message}`);
}

/**
 * Log a function call.
 * @param message The message to log.
 */
const call = (message) => {
    sendMessage("debugPrint", { message: `[CALL] ${message}` });
    console.log(`[CALL] ${message}`);
}

File: Blobcode/Log.swift
//
//  Log.swift
//  Codepad
//
//  Created by Maximilian Mittelhammer on 25.12.21.
//

import Foundation

func logInfo(message: String){
    print("[INFO] \(message)")
}

func logError(message: Error){
    print("[ERROR] \(message)")
}
